assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -133,7 +144,8 @@ const CartLineItemRow = ( { lineItem } ) => {
 					quantity={ quantity }
 					maximum={ getMaximumQuantity(
 						backOrdersAllowed,
-						lowStockRemaining
+						lowStockRemaining,
+						soldIndividually
```

Comment:
should we possibly emit out the whole quantity input if it’s sold individually?

Reply:
Core does but I prefer consistentcy personaly.

Reply:
I agree with MIke here. It keeps the ui consistent.